src/chatd.h

```diff
@@ -1602,12 +1643,17 @@ class DbInterface
     virtual Idx getIdxOfMsgidFromNodeHistory(karere::Id msgid) = 0;
 
     //  <<<--- Reaction methods --->>>
-    virtual std::string getReactionSn() = 0;
+    virtual const std::string getReactionSn() const = 0;
     virtual void setReactionSn(const std::string &rsn) = 0;
     virtual void cleanReactions(karere::Id msgId) = 0;
+    virtual void cleanPendingReactions(karere::Id msgId) = 0;
     virtual void addReaction(karere::Id msgId, karere::Id userId, const char *reaction) = 0;
+    virtual void addPendingReaction(karere::Id msgId, const char *reaction, std::string encReaction, uint8_t status) = 0;
```

Comment:
I think it's always better use `const std::string&` than `const char* ` as parameter.
My proposal would be
`virtual void addPendingReaction(karere::Id msgId, const std::string& reaction, const std::string& encReaction, uint8_t status) = 0;`

But it's only my opinion